libs/SmartStore/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'android-library'
 dependencies {
   compile project(':libs:SalesforceSDK')
   compile files('libs/sqlcipher.jar')
+  compile 'com.facebook.react:react-native:0.11.+'
```

Comment:
You don't need this entry here. ```SmartStore``` depends on ```SalesforceSDK```, which depends on ```react```, so it should automatically get pulled in.